modules/layout/react/lib/Layout.tsx

```diff
@@ -81,7 +82,7 @@ export default class Layout extends React.Component<LayoutProps> {
   };
 
   public render() {
-    const {children, spacing, gutter, capWidth, ...elemProps} = this.props;
+    const {children, gutter, capWidth, ...elemProps} = this.props;
 
     return (
       <LayoutContainer gutter={gutter} capWidth={capWidth} {...elemProps}>
```

Comment:
```suggestion
      <LayoutContainer gutter={gutter} capWidth={capWidth}>
```

I know we didn't do this before but it should be best practice that we destructure all props and then use them as needed.

Reply:
Shouldn't we be spreading the props here thought? This will fail tests for `data-propspread` attribute test

Reply:
Oof you're right. I lost track of this one, I'll re-review and try to get this one in today. So sorry about that.